main.nf

```diff
@@ -79,6 +168,7 @@ if (!(workflow.runName ==~ /[a-z]+_[a-z]+/)) {
     custom_runName = workflow.runName
 }
 
+// AWS batch settings
 if (workflow.profile.contains('awsbatch')) {
```

Comment:
Not sure this is in line with the latest template (I _think_ its not anymore 🤔 )

Reply:
Should be https://github.com/nf-core/tools/blob/ebf63095ab222e4df987c4e950ce3839e8784d3e/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/main.nf#L82-L90

Reply:
Oh, you mean the text - `// AWS batch settings`? Could add it back in?

Reply:
Done https://github.com/nf-core/tools/pull/556/commits/99a5e138e84ad3d9d8ba4f09bb98919040393153

Would be good to get your `--input` there too 😆